api/src/main/java/io/strimzi/api/kafka/model/InlineLogging.java

```diff
@@ -27,7 +27,7 @@
 
     public static final String TYPE_INLINE = "inline";
 
-    private Map<String, String> loggers = new HashMap<>();
+    private Map<String, String> loggers = new HashMap<>(0);
```

Comment:
This is fine, but it's better still to use the null pattern and allocate a `HashMap` only when there is an additional property.